backend/metasfresh-dist/metasfresh-webui-api-remote-via-ssh-tunnel.launch

```diff
@@ -1,19 +1,19 @@
+<booleanAttribute key="org.eclipse.jdt.launching.ALLOW_TERMINATE" value="false"/>
+<mapAttribute key="org.eclipse.jdt.launching.CONNECT_MAP">
+<mapEntry key="hostname" value="localhost"/>
+<mapEntry key="port" value="8789"/>
```

Comment:
our current convention is 18789, so i think you should rather update your ssh-config than the launch config